src/manage.py

```diff
@@ -139,7 +126,7 @@ def parseArgs():
     ap.add_argument("-save", "-add", action="store_true", help="Save target dump entries to local dump")
     ap.add_argument("-upd", "--update", action="store_true",
                     help="Create/update the final static.json file used by the dll from static_dump.json and static_en.json")
-    ap.add_argument("-clean", default=False, const=True, nargs="?",
+    ap.add_argument("-clean", "--clean", choices=["dump", "ui", "both"],  nargs='*', const="both", default=False,
```

Comment:
What's the reason for nargs change? Afaict this should break the clean function because it'll be passed an array.
In fact trying it out in repl, this line itself throws `ValueError: nargs must be '?' to supply const`